content/shmem_barrier.tex

```diff
@@ -38,7 +38,7 @@
 \apidescription{
     \FUNC{shmem\_barrier} is a collective synchronization routine over an
     \activeset.  Control returns from \FUNC{shmem\_barrier} after all \acp{PE} in
-    the \activeset (specified by \VAR{PE\_start}, \VAR{logPE\_stride}, and
+    the \activeset(specified by \VAR{PE\_start}, \VAR{logPE\_stride}, and
```

Comment:
Is this fixing a space issue?  If so, the correct fix is to the ```\activeset``` macro, rather than removal of the trailing space.  We can't apply this same fix in instances where the macro is followed by another word.